pipeline/report.py

```diff
@@ -416,7 +416,7 @@ def make(self, key):
         for shank in shanks:
             fig = unit_characteristic_plot.plot_driftmap(probe_insertion, shank_no=shank)
             # ---- Save fig and insert ----
-            fn_prefix = f'{water_res_num}_{sess_date}_{key["insertion_number"]}_{key["clustering_method"]}_{shank}'
+            fn_prefix = f'{water_res_num}_{sess_date}_{key["insertion_number"]}_{key["clustering_method"]}_{shank}_'
```

Comment:
not sure if we've been using f-strings yet in the codebase - this should work on our servers without updating (ubuntu 18.04 python is python3.6), not sure elsewhere..